app/src/main/java/org/p2p/wallet/swap/ui/jupiter/tokens/interactor/SwapTokensInteractor.kt

```diff
@@ -4,6 +4,7 @@ import org.p2p.wallet.home.repository.HomeLocalRepository
 import org.p2p.wallet.swap.jupiter.domain.model.SwapTokenModel
```

Comment:
why do you have interactor in `swap/ui` folder? 
Move it to `swap/interactor` or `swap/jupiter/interactor, or `jupiter/interactor`
Moreover, we don't have a folder like `domain`
Use `interactor` or `repository`

Reply:
no problem, but it's not my class and not my package
SwapTokensInteractor sits in interactor package